### PR TITLE
rm_pbs tests

### PR DESCRIPTION
**Pull request by ehiggs**
Add some test coverage to `rm_pbs.py` and actually assert some things.

(This is based off the list branch (for #71 ))

### CODE REVIEW

**Comment by ehiggs** [comment]
Jenkins: test this please.

File: hod/rmscheduler/rm_pbs.py [hunk 1]
```diff
@@ -40,7 +40,7 @@
 # optional packages, not always required
 try:
     import pbs
-    from PBSQuery import PBSQuery
+    import PBSQuery
```
Comment (boegel) [inline hod/rmscheduler/rm_pbs.py]: trailing whitespace ^_^

File: hod/rmscheduler/rm_pbs.py [hunk 2]
```diff
@@ -235,19 +230,15 @@ def info(self, jobid, types=None, job_filter=None):
             jobattr = 'NULL'
         else:
             jobattr = pbs.new_attrl(len(types))
-            for idx in range(len(types)):
-                jobattr[idx].name = types[idx]
+            for idx, name in enumerate(types):
+                jobattr[idx].name = name
 
         jobs = pbs.pbs_statjob(self.pbsconn, jobid, jobattr, 'NULL')
         if len(jobs) == 0:
```
Comment (boegel) [inline hod/rmscheduler/rm_pbs.py]: if not jobs

File: hod/rmscheduler/rm_pbs.py [hunk 3]
```diff
@@ -235,19 +230,15 @@ def info(self, jobid, types=None, job_filter=None):
...
-            res = [dict([(typ, None) for typ in types + ['id']])]  # add id
-            res = []  # return nothing
-            self.log.debug("No job found. Wrong id %s or job finished? Returning %s",
-                    jobid, res)
-            return res
-        self.log.debug("Request for jobid %s returned %d result(s) %s", jobid, len(jobs), jobs)
+            self.log.debug("No job found. Wrong id %s or job finished?", jobid)
```
Comment (boegel) [inline hod/rmscheduler/rm_pbs.py]: aren't you reintroducing a problem here, i.e. the error/warning log message?

**Comment by boegel** [comment]
@ehiggs: add a test for `master_hostname`? something like:

```python
pbs_default = os.getenv('PBS_DEFAULT')
os.environ['PBS_DEFAULT'] = 'master123.test.gent.vsc'
self.assertTrue(hrr.master_hostname(), 'master123.test.gent.vsc')
if pbs_default is not None:
    os.environ['PBS_DEFAULT'] = pbs_default
```

File: hod/rmscheduler/rm_pbs.py (hunk 3 above)
Comment (ehiggs) [inline hod/rmscheduler/rm_pbs.py]: This is a debug message. It's hard to see here, but I'm just removing the first `res = ` line which is immediately overwritten w/ an empty list. And I'm making the debug output a single line.

**Pull request merged**